pkg/store/storepb/rpc.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+package promlts;
+
+import "types.proto";
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "storepb";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_getters_all) = false;
+
+service Store {
+  rpc Series(SeriesRequest) returns (SeriesResponse) {
+    option (google.api.http) = {
+      post: "/v1/series"
+    };
+  }
+
+  rpc LabelNames(LabelNamesRequest) returns (LabelNamesResponse) {
+    option (google.api.http) = {
+      post: "/v1/labels"
+    };
+  }
+
+  rpc LabelValues(LabelValuesRequest) returns (LabelValuesResponse) {
+    option (google.api.http) = {
+      post: "/v1/labels/values"
```

Comment:
The gRPC REST gateway is always nice to have, especially for adhoc debugging with `curl`. I did what etcd did and just force everything to be a POST request, which arguably makes things simpler but less intuitive.

Any opinions?